test/test_helper.rb

```diff
@@ -20,7 +20,11 @@
 FactoryBot.use_parent_strategy = false
 
 # Do not allow network connections and external processes
-WebMock.disable_net_connect!(allow_localhost: true)
+if ENV.fetch('SELENIUM_REMOTE_HOST', nil)
+  WebMock.allow_net_connect!
```

Comment:
Can we at least scope this more narrow to the remote host?
```suggestion
  WebMock.disable_net_connect!(allow_localhost: true, allow: ENV['SELENIUM_REMOTE_HOST'])
```

It may also need to allow the right port. See https://github.com/bblimke/webmock#external-requests-can-be-disabled-while-allowing-specific-requests